### PR TITLE
Sanitize map entry title

### PR DESCRIPTION
**Pull request by hacodeorg**
# Description
Data should be sanitized before displaying in both [Volunteer map](https://code.org/volunteer/local) and [Classroom map](https://code.org/volunteer/local)

## Links
- [PLC-650](https://codedotorg.atlassian.net/browse/PLC-650)
- Follow up work: https://codedotorg.atlassian.net/browse/PLC-654

## Testing story
Please see testing details in the task.

### CODE REVIEW

**Review by islemaster — APPROVED** [review]
:+1: Let's ship this.  Will you file a PLC task to add a regression test here as well?

**Review by sureshc — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/js/volunteer-local.js [hunk 1]
```diff
@@ -429,3 +429,25 @@ function i18n(token) {
 
   return labels[token];
 }
+
+/**
+ * Replaces special characters in string by HTML entities.
+ * List of special characters is taken from
+ * https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html
+ *
+ * This is a copy of the same function from @cdo/apps/src/utils.js.
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/js/volunteer-local.js:438]: A note for posterity: Long-term, we'd like to pull this JavaScript out of pegasus and into our Webpack build.  At that point we'll want to remove this copy of `escapeHtml` and import the one from utils.js instead.  This approach is a temporary one to minimize possible impact of this change so close to Hour of Code.

**Review by islemaster — APPROVED** [review]
:+1:

File: pegasus/sites.v3/code.org/public/js/volunteer-local.js (hunk 1 above)
Comment (hacodeorg) [reply pegasus/sites.v3/code.org/public/js/volunteer-local.js:438]: Tracked by https://codedotorg.atlassian.net/browse/PLC-654

**Review by sureshc — APPROVED** [review]

**Pull request merged**